src/main/java/org/frankframework/frankdoc/model/FrankDocModel.java

```diff
@@ -134,13 +134,13 @@ protected void handle(DigesterRule rule) throws SAXException {
 			String registerTextMethod = rule.getRegisterTextMethod();
 			if(StringUtils.isNotEmpty(rule.getRegisterMethod())) {
 				if(StringUtils.isNotEmpty(registerTextMethod)) {
-					log.warn("digester-rules.xml, role name {}: Have both registerMethod and registerTextMethod, ignoring the latter", pattern.getRoleName());
+					log.error("digester-rules.xml, role name {}: Have both registerMethod and registerTextMethod, ignoring the latter", pattern.getRoleName());
```

Comment:
Dit lijkt me geen reden om de build te laten falen

Reply:
Hij is weer een warning.